artemis-chart/templates/backend-deployment.yaml

```diff
@@ -155,6 +155,16 @@ spec:
             configMapKeyRef:
               name: {{ .Release.Name }}-configmap
               key: monTimeoutLastBgpUpdate
+        - name: HIJACK_LOG_FIELDS
+          valueFrom:
+            configMapKeyRef:
+              name: {{ .Release.Name }}-configmap
+              key: hijackLogFields
+        - name: ARTEMIS_WEB_HOST
```

Comment:
I now use the artemisWebHost from configmap, which takes its value from .Values.ingress.host.

Since I do not have the setup to test that this env variable is propagated properly, could you double-check and resolve the comment?